jetty-unixsocket/jetty-unixsocket-server/src/main/config-template/etc/jetty-unixsocket-http2c.xml

```diff
@@ -6,8 +6,8 @@
     <Arg>
       <New class="org.eclipse.jetty.http2.server.HTTP2CServerConnectionFactory">
         <Arg name="config"><Ref refid="unixSocketHttpConfig"/></Arg>
-        <Set name="maxConcurrentStreams" property="jetty.http2c.maxConcurrentStreams"/>
-        <Set name="initialStreamRecvWindow" property="jetty.http2c.initialStreamRecvWindow"/>
+        <Set name="maxConcurrentStreams"><Property name="jetty.http2c.maxConcurrentStreams" default="1024"/></Set>
+        <Set name="initialStreamRecvWindow"><Property name="jetty.http2c.initialStreamRecvWindow" default="65535"/></Set>
```

Comment:
@sbordet ditto - can these be defaulted just in the code?

Reply:
Same.

Reply:
Removed xml defaults in favour of using code defaults instead.